CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseRequestHandler.swift

```diff
@@ -74,12 +74,7 @@ public class DatabaseRequestHandler {
         case "database_exists":
             let name: String = args.get(name:"name")!
             let directory: String? = args.get(name:"directory")!
-           
-            if let directory = directory {
```

Comment:
why this condition removed?

Reply:
the else condition will never be executed as the path it receives are directory and if we provide path to sqlite file, api will return false. I've tested both the scenario. If you have any scenario which requires it we can make changes accordingly.

Reply:
sent message to you

Reply:
only one signature available for "exist" method, hence the change.